code/modules/client/preferences/paint_color.dm

```diff
@@ -0,0 +1,13 @@
+/datum/preference/color/paint_color
+	savefile_key = "paint_color"
+	savefile_identifier = PREFERENCE_CHARACTER
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+
+/datum/preference/color/paint_color/is_accessible(datum/preferences/preferences)
+	if (!..(preferences))
+		return FALSE
```

Comment:
We don't mutate `preferences` at all, so this doesn't seem necessary... Shouldn't the following work?

```suggestion
/datum/preference/color/paint_color/is_accessible(datum/preferences/preferences)
	. = ..()
	if (!.)
		return FALSE
```

Calls to parent automatically feed in the arglist iirc.

Reply:
That does seem to work, thank you.

Reply:
They all do it for explicitness and because we never benefit from `.` here, plz dont do this change

Reply:
I think it was done as its own commit so shouldn't be hard to revert